sites/healthcarepackaging/server/templates/content/company.marko

```diff
@@ -0,0 +1,184 @@
+import { getAsArray, get } from "@base-cms/object-path";
+import hierarchyAliases from "@base-cms/marko-web/utils/hierarchy-aliases";
+import queryFragment from "../../graphql/fragments/content-list";
+
+$ const { site } = out.global;
+$ const { id, type, pageNode } = data;
+
+<marko-web-content-page-layout id=id type=type>
```

Comment:
Move this to common as well (`common/templates/content/company.marko`), and update the route import.